ts/@types/react-native-barcode-builder.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module 'react-native-barcode-builder' {
+    import { PureComponent } from "react";
+
+    export interface BarcodeProps {
+        value: string;
+        width?: number;
+        height?: number;
+        background?: string;
+        format: string;
```

Comment:
Format is not a simple string

Reply:
A more detailed type definition is introduced within https://github.com/teamdigitale/io-app/pull/1139/commits/36d1d0ae27f47b9b94e7271fea218c93396a6016